_source/handbook/07_installing.md

```diff
@@ -9,7 +9,15 @@ Turbo can either be referenced in compiled form via the Turbo distributable scri
 
 ## In Compiled Form
 
-You can float on the latest release of Turbo using a CDN bundler like Skypack. See <a href="https://www.skypack.dev/view/@hotwired/turbo">https://www.skypack.dev/view/@hotwired/turbo</a> for more details. Or <a href="https://unpkg.com/browse/@hotwired/turbo@latest/dist/">download the compiled packages from unpkg</a>.
+You can float on the latest release of Turbo using a CDN bundler like jsDelivr. Just include a `<script>` tag in the `<head>` of your application:
+
+```html
+<head>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@hotwired/turbo@latest/dist/turbo.es2017-esm.min.js"></script>
```

Comment:
Would encouraging consumers to use `ga.jspm.io` be appropriate?

When I execute `bin/importmap pin @hotwired/turbo@latest` in a Rails project using  Importmaps, it adds the following entry to `config/importmap.rb`

```ruby
pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.3.0/dist/turbo.es2017-esm.js"
```

Reply:
I don't have any strong preference for any CDN, as long as they work. I think it's ok to mix and match, we're not officially endorsing any.